src/Preprocessor.zig

```diff
@@ -851,6 +856,66 @@ fn stringify(pp: *Preprocessor, tokens: []const Token) !void {
     try pp.char_buf.appendSlice("\"\n");
 }
 
+fn reconstructIncludeString(pp: *Preprocessor, param_toks: []const Token) !?[]const u8 {
+    const char_top = pp.char_buf.items.len;
+    defer pp.char_buf.items.len = char_top;
+    var params = param_toks;
+
+    // Trim leading/trailing whitespace
+    if (params[0].id == .macro_ws) params = params[1..];
+    if (params.len > 0 and params[params.len - 1].id == .macro_ws) params = params[0 .. params.len - 1];
```

Comment:
`macro_ws` being non consecutive is not guaranteed, it might be true but I haven't checked it, better to just ignore them here and trim at the end.

Reply:
I ended up adjusting the trimming code to operate on tokens instead of the string since that makes it easier to detect multiple arguments outside of `<>` (e.g. pasted strings) and issue an error.